install.py

```diff
@@ -171,6 +173,9 @@ def main():
     hdf5 = "the path to the HDF5 libraries directory"
     parser.add_argument('--hdf5_root', help=hdf5)
 
+    cyclus = "the relative path to Cyclus installation directory"
+    parser.add_argument('--cyclus-root', '--cyclus_root', help=cyclus)
```

Comment:
That is ancillary. We just want the install.py files to be the same everywhere. It is too much work to maintain multiple versions, and so the version we get is the superset of all required features.